libs/model/src/models/referral.ts

```diff
@@ -12,10 +12,17 @@ export const Referral = (
   sequelize.define<ReferralInstance>(
     'Referral',
     {
-      id: {
-        type: Sequelize.INTEGER,
+      namespace_address: {
```

Comment:
We should put `referrer_address` first in the model/table otherwise the query in `GetUserReferrals.query.ts` won't use the primary key index.